src/Backoffice/Tasks/App/Controllers/GetTaskController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Tasks\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Lightit\Backoffice\Tasks\App\Transformers\TasksTransformer;
+use Lightit\Backoffice\Tasks\Domain\Actions\GetTaskAction;
+
+class GetTaskController
+{
+    public function __invoke(int $id, GetTaskAction $action): JsonResponse
+    {
+        $task = $action->execute($id);
```

Comment:
```suggestion
    public function __invoke(Task $task): JsonResponse
    {
```
Use [route model binding](https://laravel.com/docs/11.x/routing#implicit-binding).